src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document